lib/reporter.js

```diff
@@ -151,8 +155,9 @@ class CucumberReporter {
     handleAfterScenarioEvent (event, callback) {
         const scenario = event.getUri ? event : event.getPayloadItem('scenario')
         this.emit('suite:end', {
+            uid: this.getUniqueIdentifier(scenario),
             title: scenario.getName(),
-            parent: this.runningFeature.getName(),
+            parent: this.getUniqueIdentifier(this.runningFeature),
```

Comment:
The parent is no longer referred to by it's name, instead we now use the CID as well.
This is not displayed to the user.

Please see  https://github.com/webdriverio/webdriverio/pull/1717/files